<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConnect</class>
 <widget class="QDialog" name="FormConnect">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>171</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>294</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>294</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPort">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>4000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtIP">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 85 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Server IP adress :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 85 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Port :</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkVel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 85 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Velocities</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkDens">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 85 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Densities</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormConnect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormConnect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
